### DISCUSSION TITLE
Remap key to follow a link

### DISCUSSION THREAD

**Discussion by trebestie** [gdh1995/vimium-c#1126] [Q&A]
Sorry for the very dumb question,
I have to move the focus to an input field labelled 'AH' by the 'f' command
I can't find out a way to map a key which types the sequence 'f'  'ah'

My regards

**Comment by gdh1995** [comment]
If there're only a few inputs, then you may use `focusInput` to select inputs, and `Tab` and `Shift+Tab` can switch focus among them.

If you do want to select a known input, you may use `LinkHints.click` with `match=".selector"`, like
```
# select the query input box on a Google search result page
map Fah LinkHints.click match="[name=q]"
```
